doc/source/gettingstarted/installation.rst

```diff
@@ -64,47 +81,37 @@ You can also build **freud** in place so that you can run from within the folder
 Building **freud** in place has certain advantages, since it does not affect your Python behavior except within the **freud** directory itself (where **freud** can be imported after building).
 Additionally, due to limitations inherent to the distutils/setuptools infrastructure, building extension modules can only be parallelized using the build_ext subcommand of setup.py, not with install.
 As a result, it will be faster to manually run build_ext and then install (which normally calls build_ext under the hood anyway) the built packages.
-In general, the following options are available for setup.py in addition to the standard setuptools options (notes are included to indicate which options are only available for specific subcommands such as build_ext):
+
+CMake Options
++++++++++++++
+
+The scikit-built tool allows setup.py to accept three different sets of options separated by ``--``, where each set is provided directly to scikit-build, to CMake, or to the code generator of choice, respectively.
+For example, the command ``python setup.py build_ext --inplace -- -DCOVERAGE=ON -G Ninja -- -j 4`` tell scikit-build to perform an in-place build, it tells CMake to turn on the ``COVERAGE`` option and use Ninja for compilation, and it tells Ninja to compile with 4 parallel threads.
+For more information on these options, see the `scikit-build docs <scikit-build.readthedocs.io/>`__.
+
+In addition to standard CMake flags, the following CMake options are available for **freud**:
 
 .. glossary::
 
-    -\\-PRINT-WARNINGS
-      Specify whether or not to print compilation warnings resulting from the build even if the build succeeds with no errors.
+    --COVERAGE
```

Comment:
This may need a `\\`, I noticed this pattern below.
```suggestion
    -\\-COVERAGE
```

Reply:
Actually that pattern is wrong, I'm not sure why it got introduced. It renders a backslash in between the two dashes, [see the docs right now](https://freud.readthedocs.io/en/stable/gettingstarted/installation.html). If you'd like, I can go through and fix that as part of this PR.

Reply:
I fixed it.

Reply:
For posterity, this is the escaping behavior for smart double-dashes in docutils. https://docutils.sourceforge.io/docs/user/smartquotes.html#id5